### ISSUE TITLE
Aria-live?

### ISSUE THREAD

**Issue by laryn** [backdrop-contrib/fullcalendar_views#38]
I'm testing this calendar on a dev site and an internal accessibility review came back with the following:

> Interacting with the month/week, and the arrows does not announce to a user of changes: aria-live should probably be utliized.

This may be a FullCalendar library issue more than a `fullcalendar_views` issue, but flagging here in case there's something in the implementation that can be tweaked in regards to accessibility.

**Comment by indigoxela** [comment]
@laryn you may be right, it's rather a library task. They do care, lots of issues in their queue re A11Y: https://github.com/fullcalendar/fullcalendar/issues?q=is%3Aissue+is%3Aopen+label%3AAccessibility

Re aria-live... not sure, if that would be the proper thing in this case...

The buttons (proper `button` elements) have the `aria-pressed` attribute, which switches to true, if they got pressed, so in theory that should work as announcement of interaction. But I'm not sure, how assistive tools like screen readers handle that attribute.

Maybe open an issue in the libraries queue?

**Comment by laryn** [comment]
Thanks @indigoxela.
Posted for discussion in the fullcalendar queue.
